samples/ClassroomLabs/Scripts/Readme.md

```diff
@@ -8,4 +8,5 @@ This folder contains scripts to help set up classes in Azure Lab Services.
 4. [EthicalHacking](./EthicalHacking/): creates and setup an Azure Lab Services lab account that is configured to run an [ethical hacking class](https://docs.microsoft.com/en-us/azure/lab-services/classroom-labs/class-type-ethical-hacking).
 5. [HyperV](./HyperV/): helps preparing a template virtual machine for a classroom lab to use Hyper-V nested virtualization.
 6. [X2GoRemoteDesktop](./X2GoRemoteDesktop/): These scripts install the X2Go server and client for the selected Linux desktop environment (MATE, XFCE4, or Xubuntu) on your Linux lab VM. 
-7. [ActiveDirectoryJoin](./ActiveDirectoryJoin): These scripts will join the lab VMs to the domain.
\ No newline at end of file
+7. [ActiveDirectoryJoin](./ActiveDirectoryJoin): These scripts will join the lab VMs to the domain.
+8. [CustomContribRole](./CustomContribRole): This script imports a custom role that you can assign teachers to have finer control over the management operations that they can perform for a lab.
```

Comment:
Is there a reason the section is CustomContribRole instead of CustomContributorRole?  It's not a big deal, but it through me off for a second, especially after reading your blog post first.